### ISSUE TITLE
Pretendo Support - It's Time!

### ISSUE THREAD

**Issue by wicorn29** [MCMi460/3DS-RPC#31]
Nintendo has officially shut down their servers. This is a gentle reminder to add pretendo support!

**Comment by Preloading** [comment]
Hi there!

1. Duplicate of #10, #23 and #29
2. It's done, we are just waiting on MCMi460 to review my pr (#28), and for pretendo to fix a bug with there systems, the code should be mostly finalized
3. Friends still works on Nintendo Network! If you had friends, or if you friend someone, like the 3ds-rpc bot, it'll work fabulously.

**Comment by wicorn29** [comment]
Thank you. Fabulous work by the way!

**Comment by AAMBIENTT** [comment]
> Hi there!
> 
>     1. Duplicate of [Initial support for Third-Party Nintendo Network servers : Pretendo #10](https://github.com/MCMi460/3DS-RPC/issues/10), [[Request] Add Pretendo support #23](https://github.com/MCMi460/3DS-RPC/issues/23) and [Add Pretendo Support  #29](https://github.com/MCMi460/3DS-RPC/issues/29)
> 
>     2. It's done, we are just waiting on MCMi460 to review my pr ([Add Pretendo Support #28](https://github.com/MCMi460/3DS-RPC/pull/28)), and for pretendo to fix a bug with there systems, the code should be mostly finalized
> 
>     3. Friends still works on Nintendo Network! If you had friends, or if you friend someone, like the 3ds-rpc bot, it'll work fabulously.

when will this be added to the site?

**Comment by HotaruBlaze** [comment]
3ds stuff is in a messy place right now, theirs alot of issues around the bot currently, aswell as pretendo being delayed
